code/game/objects/items.dm

```diff
@@ -78,6 +78,7 @@ GLOBAL_DATUM_INIT(welding_sparks, /mutable_appearance, mutable_appearance('icons
 	/// Since any item can now be a piece of clothing, this has to be put here so all items share it.
 	/// This flag is used to determine when items in someone's inventory cover others. IE helmets making it so you can't see glasses, etc.
 	var/flags_inv
+	/// The item's DYE key, used for dyeing and machine washing purposes
 	var/item_color
```

Comment:
Sadly this isn't all this var is used for. Its also used for bullshit like e-swords.

Reply:
fixed